src/Components/Pages/SingIn/index.jsx

```diff
@@ -20,7 +19,7 @@ const SignIn = () => {
     event.preventDefault();
     setFormState({ username: '', password: '', });
     setSubmit(false);
-    console.log(`Form Submit`)
+    console.log('[SingIn] - ',formState)
```

Comment:
Actually all debug info should be excluded from the code. unless it described some error or exceptional case.

Reply:
Ok, deleted